src/Knp/Menu/Twig/MenuExtension.php

```diff
@@ -16,8 +16,11 @@ class MenuExtension extends AbstractExtension
     private $matcher;
     private $menuManipulator;
 
-    public function __construct(Helper $helper, MatcherInterface $matcher = null, MenuManipulator $menuManipulator = null)
-    {
+    public function __construct(
+        Helper $helper,
```

Comment:
I would keep signatures on one line, following the Symfony coding standards.

Reply:
Symfony itself is using a multiline constructor when the line is too long. See for example https://github.com/symfony/console/blob/5.x/Output/TrimmedBufferOutput.php

Reply:
I'm almost sure this one escaped code review, because we rejected such multiline signatures thousands of times.